ada_feeding_perception/config/food_on_fork.yaml

```diff
@@ -0,0 +1,19 @@
+# NOTE: You have to change this node name if you change the node name in the launchfile.
+food_on_fork:
+  ros__parameters:
+    # Top-left point's x-val of the bounding box rectangle
+    left_top_corner_x: 297
+    # Top-left point's y-val of the bounding box rectangle
+    left_top_corner_y: 248
+    # Bottom-right point's x-val of the bounding box of the rectangle NOTE: previously, was 422 (but 425 is better ratio)
+    right_bottom_corner_x: 425
+    # Bottom-right point's y-val of the bounding box of the rectangle
+    right_bottom_corner_y: 332
+    # minimum depth to consider (note that 330 is approx distance to the fork tine)
+    min_dist: 310
+    # maximum depth to consider (note that 330 is approx distance to the fork tine)
+    max_dist: 370
+    # absolute path for the model
+    model_location: "/home/atharva2/atharvak_node_ws/src/ada_feeding/ada_feeding_perception/model/categorical_naive_bayes_without_hand_8-30-23.pkl"
```

Comment:
This should definitely be changed from your local file system. See `face_detection` for an example. We pass in the directory of the package's share folder using the launchfile, and then the model location can be specified relative to that. That way, the code can work regardless of the system.